client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -581,8 +618,24 @@ class WpcomChecklistComponent extends PureComponent {
 	};
 
 	renderSiteLaunchedTask = ( TaskComponent, baseProps, task ) => {
-		const { needsVerification, siteIsUnlaunched, translate } = this.props;
+		const {
+			needsVerification,
+			needsEmailVerification,
+			needsDomainVerification,
+			siteIsUnlaunched,
+			translate,
+		} = this.props;
 		const disabled = ! baseProps.completed && needsVerification;
+		let noticeText;
+		if ( disabled ) {
+			if ( needsDomainVerification ) {
+				noticeText = translate(
+					'Verify email address for your domain before launching your site.'
```

Comment:
I think this should be "Verify the email address for your domain before launching your site."